docs/install/macos.markdown

```diff
@@ -24,23 +24,22 @@ export PATH
 
 ## Homebrew
 
-WezTerm is available for [brew](https://brew.sh/) users in a tap:
+WezTerm is available for [brew](https://brew.sh/) users:
 
 ```console
-$ brew tap wez/wezterm
-$ brew install --cask wez/wezterm/wezterm
+$ brew install --cask wezterm
 ```
 
 If you'd like to use a nightly build:
 
 ```console
-$ brew install --cask wez/wezterm/wezterm-nightly
+$ brew install --cask wezterm-nightly
 ```
 
 to upgrade to a newer nightly (normal `brew upgrade` will not upgrade it!):
 
 ```console
-$ brew upgrade --cask wez/wezterm/wezterm-nightly --no-quarantine --greedy-latest
+$ brew upgrade --cask wezterm-nightly --no-quarantine --greedy-latest
```

Comment:
How often do they update the cask? Or is it just pointed at the one in the tap?

Reply:
From that output, it is downloading from wezterm's nightly binary release, so it looks current